Tutorial_Making_Dataset_France_Greece.R

```diff
@@ -126,4 +123,94 @@ View(France_Host_reviews)
 #Well done you just created 2 datasets. One called Greece_Host_reviews and one called France_Host_reviews -> Checking origin variable errors!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 #Step 10 Possible in the future to combine the Greece and France dataset. By using a simular principle of creating the country datasets
 
-#Step 11 -> in the future make step 1 t/m 8 a function. Simular proces is done again.
+
+
+
+
+######################### FUNCTION 1##################################
+
+download_and_process_airbnb_data <- function(url, region, country) {
+  # Creating a variable of the URL code
+  file_name <- paste(region, "_data.csv", sep = "")
+  
+  # Download the file from the relevant URL
+  if (!file.exists(file_name)) {
+    download.file(url, file_name)
+  }
+  
+  # Decompress the dataset if it is a csv.gz to a csv file
+  if (grepl(".gz", file_name)) {
+    system(paste("gzip -d", file_name))
+    file_name <- gsub(".gz", "", file_name)
+  }
+  
+  # Read data with the read_csv function
+  data <- read_csv(file_name)
+  
+  # Create a new csv file with all "host" and "review" variables
+  selected_data <- select(data, starts_with("host"), starts_with("review"))
+  write.csv(selected_data, paste(region, "_Host_Review_data.csv", sep = ""), row.names = TRUE)
+  
+  # Add the variables "Region" and "country"
+  full_data <- mutate(selected_data, Region_Dataset = region, Country_Dataset = country)
+  
+  # Return the modified data
+  return(full_data)
+}
+
+# EXAMPLE for Greece datasets...
+athens_data <- download_and_process_data(url_Athens, "Athens", "Greece")
```

Comment:
Hey Rolf, 

As we discussed this function name you use is wrong. It misses _Airbnb, as you used before.

Kind regards,

Julian Peters

Reply:
Done!

Reply:
Hey Rolf,

It works! Exactly what we wanted to create and way faster this way. I will merge the pull request.
The only thing we need to save from the tutorial is how we make the URL's, otherwise it won't work of course. 

Kind regards,

Julian Peters